src/ray/gcs/tables.h

```diff
@@ -183,7 +190,7 @@ class TableInterface {
 /// Example tables backed by Log:
 ///   TaskTable: Stores Task metadata needed for executing the task.
 template <typename ID, typename Data>
-class Table : private Log<ID, Data>, public TableInterface<ID, Data> {
+class Table : public Log<ID, Data>, public TableInterface<ID, Data> {
```

Comment:
Not much reason besides the styleguide prefers to always use public inheritance :)

Reply:
Hmm, as I said in the original PR, I don't think it makes sense to follow the style guide in this case. `public` actually seems worse since it exposes methods that shouldn't be called on a `Table` at all. If you have a reason other than style, I'm open to this change, but that alone shouldn't be the reason for making it `public`.